src/main/java/com/tratif/pdfgen/document/builders/PageBuilder.java

```diff
@@ -0,0 +1,62 @@
+package com.tratif.pdfgen.document.builders;
+
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.Context;
+import org.thymeleaf.templateresolver.StringTemplateResolver;
+
+import java.io.StringWriter;
+import java.util.Map;
+
+public class PageBuilder {
+
+    private static TemplateEngine TEMPLATE_ENGINE;
+    static {
+        TEMPLATE_ENGINE = new TemplateEngine();
+        TEMPLATE_ENGINE.setTemplateResolver(new StringTemplateResolver());
+    }
+
+    private DocumentBuilder documentBuilder;
```

Comment:
how PageBuilder is related to the documentBuilder? is documentBuilder a parent builder? if so, name the field to make it explicit